manifests/charts/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -12,8 +12,6 @@
     defaultConfig:
       {{- if .Values.global.meshID }}
       meshId: {{ .Values.global.meshID }}
-      {{- else if .Values.global.trustDomain }}
```

Comment:
Can you check the code (DefaultMeshConfig) and verify that if meshID is not set, we default to trustDomain ?